### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by prasadjivane**
Signed-off-by: Prasad Jivane <prasadjivane@gmail.com>

Please make sure that you follow instructions from [CONTRIBUTING](https://github.com/cncf/devstats/blob/master/CONTRIBUTING.md)

Specially:
- Check if all tests pass, see [TESTING](https://github.com/cncf/devstats/blob/master/TESTING.md) for details.
- Make sure you've added test coverage for new features/metrics.
- Make sure you have updated documentation.
- If you added a new metric, please ensure you have followed instructions about [adding new metric](https://github.com/cncf/devstats/blob/master/METRICS.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ DevStats is written in [Go](https://go.dev), it uses [GitHub archives](https://w
 
 Under the hood, DevStats uses the following CNCF projects:
 - Helm (for deployment).
-- containerd (as a Kubernetes container runtime, CRI).
+- containerized (as a Kubernetes container runtime, CRI).
```
Comment (lukaszgryglicki) [inline README.md]: No, `containerd` is a program name.

**Review by lukaszgryglicki — APPROVED** [review]
Please see on emy comment.

**Comment by prasadjivane** [comment]
> Please see on emy comment.

Done..!!

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**